python-checks/src/test/resources/checks/hotspots/cors/cors.py

```diff
@@ -93,14 +93,14 @@ def flask_response_headers():
 
 def werkzeug_headers():
     from werkzeug.datastructures import Headers
-    Headers({"Access-Control-Allow-Origin": "*"}) # Noncompliant
-#           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+    Headers({"Access-Control-Allow-Origin": "*"}) # FN
```

Comment:
FQN seems to be `werkzeug.datastructures.headers.Headers` instead of expected `werkzeug.datastructures.Headers`